README.md

```diff
@@ -1,24 +1,99 @@
+  - [Vendors](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/components/mach_vendor.py#L42-L43) the latest version from upstream by calling `./mach vendor path/to/moz.yaml`
+  - [Checks the result](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L159-L171)
+  - [Commits](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/components/hg.py#L14-L19) the change locally
+  - [Submits it to try](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/apis/taskcluster.py#L51)
+  - [Comments on the bug](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L179)
+  - Finally, the new revision is [recorded in the Updatebot database](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L181).
+- That database lives in Google CloudSQL.  There is a dev and prod database, as well as [dev and prod credentials](https://searchfox.org/mozilla-central/rev/83e67336083df9f9a3d1e0c33f2ba19703d57161/taskcluster/docker/updatebot/run.py#79-84) for those databases, bugzilla, try server, phabricator, sentry, and sql-proxy (which is used to connect to the database).  You can find them in [grants.yml](https://hg.mozilla.org/ci/ci-configuration/file/tip/grants.yml#l644) searching for 'updatebot'.  The dev credentials are granted to holly, which is our reserved development instance because Updatebot can't tested on try safely.  The prod credentials are only available to mozilla-central.
+- The next time the Updatebot job runs it will get to step (6) and see that it has seen the (new) revision before.  It will [process the job from there](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L238).
+  1. Will check if all the jobs in the try run are done. If they are not, it will do nothing and check again on the next Updatebot run.
+  2. If they are done, it will look to see if there any test failures. If so it will [retrigger them](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L384) and wait until the next job.   
+  3. Once we've [received the retrigger results](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L404) then we [look at failures)(https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L411-L431), summarize them, and [add a comment to the bug and update the database](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L420-L431).
```

Comment:
This was definitely an error on my part, but `[look at failures)` should be formatted as `[look at failures]` sorry about that!